omod/src/main/webapp/resources/scripts/emr.js

```diff
@@ -87,6 +87,22 @@ var emr = (function($) {
             }
         },
 
+        serverValidationErrorMessage: function(response) {
+        	var errorMessage = "";
+        	if (response.data != null && response.data.error != null) {
+        		errorMessage = response.data.error.message;
+        	}
+        	return errorMessage;
+        },
+
+        serverGeneralErrorMessage: function(response) {
+        	var errorMessage = "";
+        	if (response.data != null && response.data.error != null) {
```

Comment:
Check if there is a non blank message field and display it otherwise you display the generic message